viahtml/templates/viahtml/blocked_page.html.jinja2

```diff
@@ -42,20 +42,20 @@
 {% endmacro %}
 
 {% block body_attrs %}
-    {% if reason.value == "malicious" %}
```

Comment:
These are gone as it's not an Enum anymore, which is a shame. It would be nice if we could emit a client directly from the git repo of Checkmate